core/tests/widgets/test_webview.py

```diff
@@ -77,6 +77,19 @@ def test_webview_load_disabled(monkeypatch):
         del DummyWebView.SUPPORTS_ON_WEBVIEW_LOAD
 
 
+def test_init_widget_with_content():
+    """Static HTML content can be loaded into the page at instantiation time."""
+    webview = toga.WebView(url="https://example.com", content="<h1>Hello, World!</h1>")
+
+    assert_action_performed_with(
+        webview,
+        "set content",
+        root_url="https://example.com",
+        content="<h1>Hello, World!</h1>",
+    )
+    assert widget.url == "https://example.com"
```

Comment:
My previous review said:

> These two cases should be up around L55, after test_create_with_values - and should be named to match.

Also - the tests should *pass* before you request a review. This test has a fairly glaring error in it.